### ISSUE TITLE
Autogenerated settings.py leads to a startup error when executing "docker-compose up".

### ISSUE THREAD

**Issue by Crysion** [mistio/mist-ce#945]
I have noticed that the first line of settings.py is the originator of the error. 
Afterwards I commented out this line for testing and then mist-ce was started successfully. 
Old config: """"User defined settings"""""
New config: #User defined settings

**Comment by kiplandiles** [comment]
Being that the file type is py I would assume that either would be accepted since that hash is definitely a comment but the triple quotes are handled as docStrings in Python and immediately garbage collected. I just installed mist.io today and had no issues with the settings.py file.

I do know it's being read in by Python as another user uncommented some items under MAILER_SETTINGS but forgot to remove the space at the beginning. Mist.io would not start because of improper indentation.

Im running on Ubuntu 18.04.1 LTS.

I have noticed that mist.io is resource intensive and it would not run well or consistently in Docker on my MacBook with only 4GB or RAM allocated to Docker. Once I moved it to bare metal with 8 cores and 16Gb or Ram everything seemed to work much better - although - still see some sporadic screen refresh issues. Load on that box hovers at 1.8 so its working pretty hard (although 1.8 on a 8 core system is just fine).
